patches/net-traffic_annotation-BUILD.gn.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/net/traffic_annotation/BUILD.gn b/net/traffic_annotation/BUILD.gn
+index 771970a0462592317bcc061c7763eb55e67458df..66e73e69e4cf1e79ece8f49f59ad0c30f87647b2 100644
+--- a/net/traffic_annotation/BUILD.gn
++++ b/net/traffic_annotation/BUILD.gn
+@@ -8,7 +8,7 @@ source_set("traffic_annotation") {
+ }
+ 
+ source_set("test_support") {
+-  testonly = true
++#  testonly = true
```

Comment:
the chromedriver one is not good, but if we need to comment out `testonly` for `test_support` then we have a much more serious dependency issue to address

Reply:
This one is pulled in by `chromedriver`:
<pre>
ERROR at //build/config/BUILDCONFIG.gn:565:5: Test-only dependency not allowed.
target(_target_type, target_name) {
^----------------------------------
//chrome/test/chromedriver:chromedriver
which is NOT marked testonly can't depend on
//net/traffic_annotation:test_support
which is marked testonly. Only targets with "testonly = true"
can depend on other test-only targets.
</pre>

Reply:
@bridiver this is used for chromedriver... we aren't packaging that with the build. Do we need to be including this when doing `create_dist`?

Reply:
d4cf829